src/Controller/EnMarche/AdherentController.php

```diff
@@ -38,13 +41,32 @@ class AdherentController extends Controller
      */
     public function homeAction(Request $request): Response
     {
+        $nbAdherent = $this->getDoctrine()->getRepository(Adherent::class)->count(['adherent' => true]);
```

Comment:
Use `countAdherents` instead please.

Reply:
This can put moved directly in the `render` method.

Reply:
Don't use doctrine registry and instead use Action Injection feature of controllers.
E.g. add `AdherentRepository $adherentRepository` parameter in the action method.